### DISCUSSION TITLE
Ticks callback support

### DISCUSSION THREAD

**Discussion by mjhillintel** [ipax77/pax.BlazorChartJs#30] [Q&A]
Is there a way currently, or any plans to support, the ticks callbacks?  Specifically, I'm interested in applying a custom formatting function for the tick labels.  Something similar to what described here: https://www.chartjs.org/docs/latest/axes/labelling.html.  
Thanks for any info.

**Comment by ipax77** [comment]
For now you have to bite the bullet and add your own custom JavaScript. It might be possible to add the functionality of callbacks using eval, but it would be ugly and would have [serious security concerns](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#never_use_eval!). 

I added a [sample chart](https://ipax77.github.io/pax.BlazorChartJs/callbackchart) for the tick callback using custom JavaScript.
[Component](https://github.com/ipax77/pax.BlazorChartJs/blob/master/src/pax.BlazorChartJs.samplelib/EventcallbackChartComp.razor)
[JavaScript](https://github.com/ipax77/pax.BlazorChartJs/blob/master/src/pax.BlazorChartJs.samplelib/wwwroot/callbackChart.js)

**Reply by mjhillintel** [reply]
Thanks for the quick response, and for the working examples. I'm now up and running with this and it seems like a reasonable way to go. Thanks again.
